pkg/repository/workflow_run.go

```diff
@@ -49,7 +49,8 @@ type CreateWorkflowRunOpts struct {
 	ChildKey *string
 
 	// (optional) the child index of the workflow run, if this is a child run of a different workflow
-	ChildIndex *int
+	// ChildIndex *int `validate:"omitempty,min=0"`
+	ChildIndex *int // I feel like I should be able to tighten this up but it seems a -1 for child index is coming from somewhere
```

Comment:
`self.spawn_index = -1`

tracked this down to the pythonsdk.. defaults to -1 for some reason

Reply:
`hatchet_sdk/context/context.py`